packages/UI/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  Pagination,
+  PaginationContent,
+  PaginationJump,
+  PaginationItem,
+  PaginationLink,
+} from './Pagination';
+
+const meta: Meta<typeof Pagination> = {
+  component: Pagination,
+};
+
+export default meta;
+type Story = StoryObj<typeof Pagination>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/react/api/csf
+ * to learn how to use render functions.
+ */
+
+export const Primary: Story = {
+  render: () => (
+    <Pagination>
+      <PaginationContent>
+        <div className="flex gap-2">
```

Comment:
We can remove all the divs and move the className to PaginationContent

Reply:
using something like grid? not sure what you mean here.
can you please give me an example, I don't like these divs there either!

Reply:
You can just put the className in the Pagination Content and get rid of all the divs. The divs don't really do anything in this situation, you are splitting them, but nothing special with layout with the split divs.

Reply:
the split divs actually change the layout as they group the arrows left, numbers, arrows right.
each div acts as a flex container that puts a gap between each element by `0.5rem` aka `gap-2`.

`<PaginationContent />` is already a flex container with gap of `2rem` aka `gap-8` so I'm not sure how I can do the same without those extra divs, here's a screenshot explaining the current implementation!
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/27841994/5befa9f7-d8d8-402e-a172-bff238e61dc4)

Reply:
Oh sorry, yeah, i didn't think that gap-8 created space between the three divs.  Ignore me then!